rsts/deployment/ideal_flow.rst

```diff
@@ -3,17 +3,21 @@
 How to Streamline Your Flyte Workflows
 --------------------------------------
 
-Flyte could be useful for many use cases, be it model training, data processing, ELT/ETL, or bioinformatics.
-When workflows are built and deployed, be it in any domain, we may need automation to reduce human-in-the-loop to some extent;
+Flyte has a wide range of applications, including model training, data processing, ELT/ETL, and bioinformatics. 
+Regardless of the domain, when workflows are built and deployed, we may need automation to reduce human-in-the-loop to some extent; 
 essentially, to automate serializing and registering workflows, creating docker containers, etc.
 
 Before diving into an example use case that explains how DevOps could power Flyte pipelines, let's look at some DevOps-style features Flyte ships with.
 
-- Every DAG can be versioned with Git commit's SHA or the hash of the code artifact; to know more about versioning, see :ref:`divedeep-versioning`
-- Tasks and workflows are immutable, i.e., a version is immutable; any mutation results in a new version
-- In the case of rapid iteration of code with no dependency modification, :ref:`fast registration <Fast Registration>` can be used
-- Multi-tenancy support through projects and domains
-- Executions can be monitored using logs on :ref:`Flyte UI <ui>`
+- Every DAG can be versioned with Git commit's SHA or the hash of the code artifact; to know more about versioning, see :ref:`divedeep-versioning`.
+- Tasks and workflows are immutable, i.e., a version is immutable, which means that any change results in a new version.
+- In the case of rapid iteration of code with no dependency modification, :ref:`fast registration <Fast Registration>` can be used.
+- Multi-tenancy support through projects and domains.
+- Executions can be monitored using logs on :ref:`Flyte UI <ui>`.
+
+.. note::
+Currently, Flyte does not support the promotion of workflows across various stages (development, staging, production). This promotion process is expected to be handled externally through CI/CD process.
+Here's an example case study.
```

Comment:
```suggestion
Currently, Flyte does not support the promotion of workflows across various stages (development, staging, production). This promotion process is expected to be handled externally through CI/CD process.
```
It doesn't make sense to have continuation between this and the next section from within a note.